client/src/components/Tags.tsx

```diff
@@ -18,9 +18,10 @@ export default function GenerateTags(props: { tags: string[], space?: number })
 }
 
 export function Tag(props: { tag: string }) {
+  const truncated = props.tag.length > 10 ? props.tag.slice(0, 7) + '...' : props.tag;
```

Comment:
Isn't the limit 3 tags?

Reply:
This is to limit the length of an individual tag which is a string, since we don't want them to be too long and overflow on a post preview.

Reply:
@danielduong8 don't truncate the tag in the post dialog, only truncate in preview. Perhaps add a prop `shouldTruncate` to help with this